src/getting-started/contribute.md

```diff
@@ -29,13 +29,15 @@ Simply click this icon to provide a rating for the current page and, optionally,
 
 ## Log an issue
 
-When you see a problem with one of our pages, please let us know. This could be a broken link, content error, or other item that you can identify for fixing.
+When you see a problem with one of our pages, such as a broken link, content error, or other item you can identify for fixing, let us know.
+
+To log an issue:
```

Comment:
We have been getting rid of the redundancy  of the "To do x" items when there is a single task in a section that already has a "Do x" heading.

Reply:
Oh, ok. I have just never seen that approach before, as lists (to my knowledge) always need an intro to guide the user.